File: pset6/mario-less/mario.py
import cs50


def main():
    while True:
        height = cs50.get_int("Height: ")
        if height > 0 and height < 9:
            break

    for i in range(1, height + 1):
        for j in range(1, height + 1):
            if j > (height - i):
                print("#", end="")
            else:
                print(" ", end="")
        print("")


main()

File: psets/pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int quantitySteps = 0;

    // Repeat until user provide a number between 0 and 9
    do
    {
        quantitySteps = get_int("Height: ");

    }
    while (quantitySteps < 1 || quantitySteps > 8);

    // Do a loop through the quantity steps representing each line
    for (int i = 1; i <=  quantitySteps; i++)
    {
        // Loop to check and print each step from line in staircase that is going up
        for (int y = quantitySteps; y >= 1; y--)
        {
            if (y <= i)
            {
                printf("#");
            }
            else
            {
                printf(" ");
            }
        }

        // Print white space to separate staircase
        printf("  ");

        // Loop to check and print each step from line in staincase that is going down
        for (int y = 1; y <= quantitySteps; y++)
        {
            if (y <= i)
            {
                printf("#");
            }
        }

        // Break the line to continue loop
        printf("\n");
    }

}